lesson05/assignment/css/_components.content.scss

```diff
@@ -1,13 +1,82 @@
-html {
-    box-sizing: border-box;
+* {
+    font-family: $font;
 }
 
-*,
-*:after,
-*:before {
-    box-sizing: inherit;
+//Style header
+.item1 {
```

Comment:
I would use more descriptive class names here, like `nav`, `aside`, `footer`. Or, just use the HTML element for the selector, since those are already the element names! We strive to use meaningful, easy to understand names in our code, to make it easier to work with.